suite-native/module-connect-device/src/screens/ConnectDeviceCrossroadsScreen.tsx

```diff
@@ -53,25 +49,31 @@ export const ConnectDeviceCrossroadsScreen = () => {
                         <Pictogram
                             icon="trezor"
                             variant="green"
-                            title={"I've got my Trezor"}
+                            title="I've got my Trezor"
                             subtitle="Connect to manage your assets"
                             size="large"
                         />
-                        <Button onPress={handleConnectDevice}>Connect Trezor</Button>
+                        <Button onPress={handleConnectDevice} size="large">
+                            Connect Trezor
+                        </Button>
                     </VStack>
                 </Card>
                 <Card style={applyStyle(cardStyle, { flex: 1 })}>
                     <VStack spacing="large" justifyContent="center" alignItems="center">
                         <VStack spacing="small" alignItems="center">
-                            <Text variant="titleSmall" style={applyStyle(textStyle)}>
+                            <Text variant="titleSmall" align="center">
                                 Sync coins without your Trezor
                             </Text>
-                            <Text color="textSubdued" style={applyStyle(textStyle)}>
+                            <Text color="textSubdued" align="center">
                                 Track your favorite coins anytime, anywhere, even when your Trezor
                                 isn't connected.
                             </Text>
                         </VStack>
-                        <Button onPress={handleSyncMyCoins} colorScheme="tertiaryElevation0">
+                        <Button
+                            onPress={handleSyncMyCoins}
+                            colorScheme="tertiaryElevation0"
```

Comment:
One more little thing. This should be actually

```suggestion
                            colorScheme="tertiaryElevation1"
```

Because it is on the `Card` component that is defined in Figma as the elevation-1. [Figma reference](https://www.figma.com/file/Z6AGVUmKQzLNtDozFamW7f/%5Bs2.0%5D-Mobile?type=design&mode=design&t=IyDw0H3fdxqYM7qm-0).

Reply:
Your link doesn't lead me anywhere, but here is screenshot with the color which is correct as `tertiaryElevation0`

<img width="1157" alt="image" src="https://github.com/trezor/trezor-suite/assets/36101761/a7af746c-fb75-408a-a389-1f4138ecbdb6">

Reply:
Opps, this is the [correct link](https://www.figma.com/file/Z6AGVUmKQzLNtDozFamW7f/%5Bs2.0%5D-Mobile?type=design&node-id=14455%3A113410&mode=design&t=iWSS7xFNTgzs31mV-1).

Reply:
Fixed [030ce08](https://github.com/trezor/trezor-suite/pull/9450/commits/030ce08d8e81cad2d1cd019ecb99f44fe24ac0b0)

But please, raise this issue with designers or @mnuky and don't waste time on this in the pull requests as we should be doing this according to design. I don't know whether there is reason for this or not and figuring this out is a waste of our time to have these discussions. We go by design and if there is a problem there, we should generally solve it in the design first.

Reply:
Plus I can see the button there also having `elevation0`
<img width="588" alt="image" src="https://github.com/trezor/trezor-suite/assets/36101761/44a6facf-c05b-481b-ad23-204f6d024b14">

Reply:
Already mentioned in the Figma design file.